### DISCUSSION TITLE
Atrasos en el curso

### DISCUSSION THREAD

**Discussion by CristianAM05** [Launch-X-Latam/MisionFrontEnd#402] [General]
Hola, tuve algunos problemas el mes pasado para iniciar el curso, recién este mes estoy tratando de ponerme al día pero ya solo quedan 2 días como límite para las prácticas.
No sé si hacer todo lo posible por terminar o ya de lleno meterme al BackEnd para no atrasarme más, es posible?

**Comment by TonatiuhDSJ** [comment]
Puedes terminar, los videos son para que veas como es la estructura más en si de como se conforma html, css y js.
Lo que puedes hacer es poner los videos de fondo, mientras haces las practicas.

**Reply by CristianAM05** [reply]
Si, eso estoy haciendo, ya termine HTML, aspiro a terminar CSS y JS entre hoy y mañana.  Una pregunta las practicas se montan en github solamente y ya?

**Reply by TonatiuhDSJ** [reply]
puedes solo ponerlas en el github o ya si quieres ponerlas en algun servicio, puedes hacerlo en netlify.

**Reply by CristianAM05** [reply]
Okey, muchas gracias
